docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -406,7 +406,7 @@ def purge_old_file_versions(
 
     # Number of versions to keep is determined by the account type
     # and by whether the user has opted-in to a project-based count
-    if project.use_storage_keep_versions:
+    if project.is_premium:
```

Comment:
As far as I remember our conversation with Marco and what I have posted on ClickUp, we should have used the plan's `is_premium` flag, not adding a `is_premium` on the project?

Reply:
Absolutely, this is a leftover

Reply:
Actually looking back at it, the agreement was:

```
project's owner's plan = community ?
- max versions = default 3 = default
else:
- max versions = `not defined`
- default = 10
```

So this is fine?